packages/node-api/src/apis/index.js

```diff
@@ -49,6 +50,8 @@ async function createApi(network, endpoint) {
     options = kintsugiOptions;
   } else if (chains.polkadex === network) {
     options = polkadexOptions;
+  } else if (chains.interlay === network) {
+    options = interlayOptions;
```

Comment:
How about:
```javascript
if ([chains.kintsugi, chains.interlay].includes(network)) {
  options = kintsugiOptions;
}
```
So we don't need the redundant `interlay.js` for options.

Reply:
I agree to remove redundancy, but I think we should remove kitsugui and use interlayOptions since Interlay is the primary network for the project.

Reply:
Fine to me.